wordpress-mobile/gutenberg-mobile
# Fix download button in File block

Fixes #https://github.com/WordPress/gutenberg/issues/28706

gb PR: https://github.com/WordPress/gutenberg/pull/28704

To test:

Add file block
Add file from device
Observe the button
Expect button opacity is lowered

PR submission checklist:
